DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/README.md

```diff
@@ -0,0 +1,354 @@
+| svd_demo.cpp           | Host code                       | Launch a demonstration using the SVD design. Contains the `main()` function.                                                                                                                         |
+| svd_testcase.hpp       | Host code                       | A `struct` that lunches the design with a set of input and check result for correctness. The input can either be specified or generated.                                                             |
+| svd_testbench_tool.hpp  | Host code                       | Helper functions that are used in the test bench of the SVD demonstration to check for correctness.                                                                                                   |
+| print_matrix.hpp       | Host code                       | Helper functions to print matrices.                                                                                                                                                                  |
+| golden_pca.hpp         | Host code                       | A CPU reference PCA design that is used to calculate reference eigenvalues for the test bench.                                                                                                      |
+| svd.hpp                | Host code with some device code | Contains wrapper function `SingularValueDecomposition` that launches individual kernels of the SVD design.                                                                                           |
+| memory_transfers.hpp   | Device code                     | Contains kernel implementations of `MatrixReadFromDDRTo2PipesByBlocks` , `MatrixReadPipeToDDR` and `VectorReadPipeToDDR`. These kernels transfers data between DDR and streaming interfaces (pipes).|
+| usv_from_eigens.hpp       | Device code                     | Contains kernel implementation of `USVFromEigens`.                                                                                                                                                   |
+### Input covariance matrix computation
+The covariance computation in this design is from `streaming_covariance_matrix.hpp` in the shared [include](../../include) directory and the parameter `standardized` is set to `false`. This source code is shared with [PCA](../pca/) reference design.
+
+The covariance of input A is equal to the transpose of A multiplied by A:
+```math
+Cov(A) = A^{T}A
+```
+Therefore this kernel performs a matrix multiplication by blocks, as described in the PCA reference design.
+
+### Eigenvalues and Eigenvectors computation
+We are reusing the `fpga_linalg::StreamingEigen` design from the PCA sample to compute eigenvalues and eigenvectors of the input covariance matrix. These eigenvalues and eigenvectors are going to be used to construct the final outputs of SVD.
```

Comment:
```suggestion
This design imports `streaming_eigen.hpp` from the shared [include]() directory. The eigenvalues and eigenvectors of the input covariance matrix are computed using the `fpga_linalg::StreamingEigen` functor. These eigenvalues and eigenvectors are going to be used to construct the final outputs of SVD.
```

use consistent language across each of these paragraphs so that the user is not surprised. The flow should be something like, "This design includes XXX. It uses YYYY functor to implement ZZZ functionality." Optionally, say "This functor is also used in AAAA design."